DataTool/ToolLogic/Extract/ExtractFlags.cs

```diff
@@ -97,6 +97,9 @@ public class ExtractFlags : ICLIFlags {
         // [CLIFlag(Default = false, Flag = "convert-bnk", Help = "Convert .bnk files to .wem", Parser = new[] { "DataTool.Flag.Converter", "CLIFlagBoolean" })]
         // public bool ConvertBnk;
 
+        [CLIFlag(Default = false, Flag = "XML", Help = "Convert STUs to xml when extracted with ExtractDebugType", Parser = new[] { "DataTool.Flag.Converter", "CLIFlagBoolean" })]
```

Comment:
Flags should be lowercase.

Reply:
nobody has ever wanted to see this on the help for normal extract modes
![zingy](https://user-images.githubusercontent.com/13888932/115040786-b0fe4100-9ec9-11eb-9904-7908929d70e2.png)